src/main/java/bio/terra/service/job/JobMapKeys.java

```diff
@@ -17,7 +17,8 @@
   EXPORT_GSPATHS("exportGsPaths"),
   FILE_ID("fileId"),
   ASSET_ID("assetId"),
-  TRANSACTION_ID("transactionId");
+  TRANSACTION_ID("transactionId"),
+  EXPORT_VALIDATE_PK_UNIQUENESS("exportValidatePkUniqueness");
```

Comment:
I think the pattern we've been using has been that there'd be an ExportMapKeys class with EXPORT_GSPATHS and EXPORT_VALIDATE_PK_UNIQUENESS